src/queries/Business.js

```diff
@@ -63,7 +65,7 @@ export default {
           business.email, business.address
         FROM businesses 
         business LEFT JOIN reviews ON reviews.business_id = business.id
-        WHERE status='approved' AND business.id=$1 
```

Comment:
let's not change it here, let's change it so everytime we add a new business it will default to `approved` instead of `pending` status